### PR TITLE
docs: add documentation for Trussels

### LINKED ISSUE #97: Security PR rotation logistics
Start a conversation and determine logistics for how we want to handle a security rotation. This function should support:
- Team members from Truss can opt-in to be part of the "on call" rotation
- On-call shifts should have a pre-determined length of time
- Add description of responsibilities while on-call -- primarily paying attention to any security alerts that come in from Github, making sure urgent PRs are being attended to, etc. The on-call person may not be required to resolve issues themselves, but instead make sure they are not being dropped or ignored.

### LINKED ISSUE #185: Add documentation for Trussel on/off-boarding
Since most of the time, Trussels who are working on this library actively will probably be on Reserve, they may need to be onboarded/offboarded relatively quickly and frequently. We may also often run into the situation where a Trussel is not able to complete work in progress if they end up joining a new client project on short notice. We should add some documentation that outlines what to do when you are onboarding & offboarding as an "active" contributor to this project.

### PR DESCRIPTION
**Pull request by suzubara**
Adds some documentation specifically for Trussels who are looking to get more or less involved in this project! I also took the opportunity to update the README following the best practices outlined at https://github.com/RichardLitt/standard-readme

Fixes #185
Fixes #97

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,30 +67,28 @@ Also make sure to include the following in order to import the compiled CSS from
...
-- [ ] Add component scaffolding shortcut (for generating component, tests, stories files with template code)
-- [ ] Add testing coverage collection
-- [ ] Set up Storybook as public Github page
-- [ ] Add example application that uses the library to the repo
-- [ ] Add visual testing automation tool (i.e., Loki)
-- [ ] Enforce adding to CHANGELOG when merging a PR into develop
-- [ ] Make sure new components are added as package exports
+## Maintainers
+
+- [@suzubara](https://github.com/suzubara)
```
Comment (suzubara) [inline README.md:72]: ***Please add yourself here as part of this PR!*** (if you read the doc about active maintainers and would like to be one right off the bat)

File: README.md [hunk 2]
```diff
@@ -56,30 +67,28 @@ Also make sure to include the following in order to import the compiled CSS from
 
 **[More info about using USWDS CSS & SCSS](./docs/scss.md)**
 
-## Roadmap
```
Comment (suzubara) [inline README.md:59]: fyi: I made sure any outstanding roadmap items were reflected in github issues before deleting this section

File: README.md [hunk 3]
```diff
@@ -56,30 +67,28 @@ Also make sure to include the following in order to import the compiled CSS from
...
+
+## Contributing
+
+Interested in contributing? See our [guidelines and dev setup here](./docs/contributing.md).
+
+Are you a Trussel and new to this project? Check out our [on & offboarding guide](./docs/for_trussels.md) made just for you!
+
+## License
+
+[Copyright 2019, TrussWorks, Inc.](../LICENSE)
```
Comment (eamahanna) [inline README.md:82]: Is this the standard licensing practice for Truss?

File: docs/for_trussels.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+If you've completed all of the above and are wondering what to do next, here are some ideas!
+
+- Check out any [open pull requests](https://github.com/trussworks/react-uswds/pulls) that might need reviews or feedback
+- Take a look at the [project board](https://github.com/trussworks/react-uswds/projects/1) to get a sense of current and upcoming work
+  - Issues in the **Ready for development** column should have clear scope and acceptance criteria, and are ready to be worked on!
+    - especially any that are labeled `good first issue` if this is your first time contributing to ReactUSWDS
+  - Issues in the **Needs requirements** column are undergoing active discussion and would probably benefit from your point of view!
+- Browse [open issues](https://github.com/trussworks/react-uswds/issues) to see if there are any new issues that need to be labeled or prioritized
+
```
Comment (eamahanna) [inline docs/for_trussels.md:50]: I think another good onboarding step would be to check in with the other active maintainers to help get a feel for where help is needed and things along those lines :) (just an opinion though)

File: docs/for_trussels.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+Regardless of whether you are on client work or Reserve, it is _up to each individual_ to delineate themselves as an active maintainer, and perform the following on- or off-boarding tasks as needed.
+
+### Active maintainer responsibilities include:
+
+- Maintain a presence in the #react-uswds channel, especially to answer any implementation questions from Trussels who are _not_ active maintainers
+- Attend and help facilitate the ReactUSDS check-in meetings regularly
+- Pay attention to [USWDS](https://github.com/uswds/uswds) updates, and create new issues needed to help this project stay up-to-date with them
+- Participate actively in PR reviews, issue discussion, and project roadmap planning
+- Keep an eye on any security alerts that come up, and make sure they are addressed promptly
```
Comment (eamahanna) [inline docs/for_trussels.md:25]: Do we have any documentation about what to do when security alerts arise and how to handle them?

File: README.md (hunk 3 above)
Comment (haworku) [reply README.md:82]: Yes I believe so. FYI there is a TDR drafted by @ferlatte I saw circulating about what licenses we use internally at Truss, goes into more detail about what licenses we should use and when.

Comment (suzubara) [reply README.md:83]: 👍 I'm not entirely sure if we have the right license but if we need to change it I'd handle that as a separate issue!

File: docs/for_trussels.md (hunk 5 above)
Comment (suzubara) [reply docs/for_trussels.md:25]: nooope, I'll add a brief note on that and maybe we can add onto it as things come up..

File: README.md [hunk 6]
```diff
@@ -1,22 +1,27 @@
...
 1. [Usage](#usage)
+1. [Maintainers](#maintainers)
+1. [Contributing](#contributing)
+1. [License](#license)
 1. [Roadmap](#roadmap)
 
-## Goals
+## Background
 
 The primary deliverable is a published npm package that can be included as a dependency in other projects that use USWDS with React. In order for these components to be actually useful, they should follow best practices and baseline standards for accessible, semantic, markup; be well-tested across browsers and devices; and allow for an appropriate level of customization in implementation (such as via React `props`). Therefore we should adhere to these development guidelines as much as possible:
```
[the remark this answers is not in the document]
Comment (suzubara) [reply README.md:26]: agreed, I could see this falling into https://github.com/trussworks/react-uswds/issues/73 ?

**Pull request merged**